packages/react-components/react-menu-grid-preview/library/src/components/MenuGridCell/useMenuGridCellStyles.styles.ts

```diff
@@ -1,15 +1,23 @@
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import { mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses } from '@griffel/react';
 import type { MenuGridCellSlots, MenuGridCellState } from './MenuGridCell.types';
 
 export const menuGridCellClassNames: SlotClassNames<MenuGridCellSlots> = {
   root: 'fui-MenuGridCell',
 };
 
+const useStyles = makeStyles({
+  root: {
+    boxSizing: 'border-box',
```

Comment:
doesn't seem to work:

<img width="147" height="65" alt="image" src="https://github.com/user-attachments/assets/9392549a-a703-4118-bd16-77872eda4fcd" />

goal is to make the row same height as the menu item: 
<img width="179" height="89" alt="image" src="https://github.com/user-attachments/assets/a230565b-ff61-4a0b-9fee-290211714c9b" />